api/src/main/java/ai/djl/translate/ServingTranslatorFactory.java

```diff
@@ -81,7 +82,7 @@ public Set<Pair<Type, Type>> getSupportedTypes() {
         Path libPath = modelDir.resolve("libs");
         if (!Files.isDirectory(libPath)) {
             libPath = modelDir.resolve("lib");
-            if (!Files.isDirectory(libPath)) {
+            if (!Files.isDirectory(libPath) && className == null) {
```

Comment:
If the "lib/libs" folder doesn't exist, we should keep searching in the classpath, so we can load the translator from "deps" folder or system classpath